project-base/src/Shopsys/ShopBundle/Resources/styles/front/common/components/box/progress.less

```diff
@@ -7,25 +7,14 @@
     width: 100%;
     margin-bottom: @margin-gap;
 
-    .is-no-flex & {
-        display: block;
-        .clearfix();
-    }
-
     &__item {
-        flex: 1;
+        width: 100%/@box-progress-steps-count;
```

Comment:
Please check if you can substitute `flex` property for the `width` property.

Reply:
Flex: 1 sets all flexible items to be same lenght. Is not necessary bcs, we set calculated width.

Reply:
It is okay for now. We will refactor this later.